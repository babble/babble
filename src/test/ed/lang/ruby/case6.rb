
def foo(a,b)
  c = 
    case a
      when 5
      puts "Y"
      a = a + 1
      puts "Y#{a}"
      b = b + 2
      else
      puts "Z#{a}"
      a = a - 1
      puts "Z#{a}"
      b = b + 1
    end
  puts a
  puts b
  puts c 
  [ a , b , c ]
end

a = foo(2 , 3 )
puts( a[0] )
puts( a[1] )
puts( a[2] )

