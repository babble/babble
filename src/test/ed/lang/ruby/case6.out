Z2
Z1
1
4
4
1
4
4
